software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java

```diff
@@ -125,11 +125,16 @@ public Void call() {
 
         Entities.dumpInfo(cluster);
 
-        KafkaSupport support = new KafkaSupport(cluster);
+        final KafkaSupport support = new KafkaSupport(cluster);
 
         support.sendMessage("brooklyn", "TEST_MESSAGE");
-        Thread.sleep(Duration.seconds(5).toMilliseconds());
-        String message = support.getMessage("brooklyn");
-        assertEquals(message, "TEST_MESSAGE");
+
+        Asserts.succeedsEventually(MutableMap.of("timeout", Duration.FIVE_SECONDS), new Runnable() {
```

Comment:
My personal preference is to not bother configuring an explicit timeout (so it defaults to 30 seconds). Given they are not performance tests, I don't see the point in having a short(ish) timeout. On low-powered asf builds we've seen some unit tests take several seconds.

Happy to merge as-is. You've improved it over the 5 second sleep, and left the time as it was. And this is an integration test, so shouldn't fail on a low-powered asf build server.